File: Quest/Assets/Src/Cards/Ally Cards/SirPercival.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class SirPercival : AllyCard
{

    public SirPercival() : base("Sir Percival", 5, 1) //base 5
    {

    }

    public override int getBattlePoints(GameState gState) //override 20
    {
        if (gState.getCurrentStoryCard().getName().Equals("Search for the Holy Grail"))
        {
            return battlePoints = 20;
        }
        return battlePoints = 5;
    }

}

File: eclipse/Quest/src/model/cards/allyCards/SirPercival.java
package model.cards.allyCards;

import model.cards.AllyCard;
import model.game.GameState;

public class SirPercival extends AllyCard {

	public SirPercival() {
		super("Sir Percival", 5, 0); //base 5
	}

    public int getBattlePoints(GameState gState) //override 20
    {
        if (gState.getCurrentStoryCard().getName().equals("Search for the Holy Grail"))
        {
            return battlePoints = 20;
        }
        return battlePoints = 5;
    }
    
}
